vortex-btrblocks/src/float/stats.rs

```diff
@@ -135,11 +135,12 @@ where
         };
     }
 
-    let validity = array.validity_mask().vortex_expect("logical_validity");
-    let null_count = validity.false_count();
-    let value_count = validity.true_count();
-    let mut min = T::max_value();
-    let mut max = T::min_value();
```

Comment:
doesn't seem like we actually do anything with min/max here, we can always add them later if that makes sense.

Reply:
oh lol, i guess clippy did't notice these b/c they are read/written even though they aren't returned at the end